modules/seqkit/pair/meta.yml

```diff
@@ -0,0 +1,44 @@
+name: seqkit_pair
+description: match up paired-end reads from two fastq files
+keywords:
+  - seqkit
+  - pair
+tools:
+  - seqkit:
+      description: Cross-platform and ultrafast toolkit for FASTA/Q file manipulation, written by Wei Shen.
+      homepage: https://bioinf.shenwei.me/seqkit/usage/
+      documentation: https://bioinf.shenwei.me/seqkit/usage/
+      tool_dev_url: https://github.com/shenwei356/seqkit/
+      doi: "10.1371/journal.pone.0163962"
+      licence: ['MIT']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: |
+        List of input paired-end FastQ files.
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - reads:
```

Comment:
`unpaired_reads` is missing

Reply:
its optional

Reply:
It should still be included here as far as I'm aware.

Reply:
I agree with @mahesh-panchal, as it can exists (as you have it as an optional output), information should be in the meta.yml file

Reply:
The outputs here are nearly all optional.
https://github.com/nf-core/modules/blob/e20e57f90b6787ac9a010a980cf6ea98bd990046/modules/mapdamage2/meta.yml